account_invoice_line_description/README.rst

```diff
@@ -1,48 +1,64 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+================================
 Account invoice line description
 ================================
 
-This module allows to use only the product description on the invoice lines.
+This module allows to use only the sale description or the purchase description
+on the invoice lines depending on the invoice type.
 
 Configuration
 =============
 
 The user has to belong to group_use_product_description_per_inv_line.
```

Comment:
I would put the non-technical name here: _Allow using only the product description on th invoice lines_